base_comment_template/models/comment.py

```diff
@@ -9,23 +9,35 @@ class BaseCommentTemplate(models.Model):
     _name = "base.comment.template"
     _description = "Base comment template"
 
+    active = fields.Boolean(default=True)
+
     name = fields.Char(
         string='Comment summary',
-        required=True
+        required=True,
     )
+
     position = fields.Selection(
         selection=[
             ('before_lines', 'Before lines'),
-            ('after_lines', 'After lines')
+            ('after_lines', 'After lines'),
         ],
         required=True,
         default='before_lines',
-        help="Position on document"
+        help="Position on document",
     )
+
     text = fields.Html(
         string='Comment',
         translate=True,
-        required=True
+        required=True,
+    )
+
+    company_id = fields.Many2one(
+        'res.company',
+        string='Company',
+        help="If set, it'll only be available for this company",
+        ondelete='cascade',
```

Comment:
Why `cascade`? I think it's better default `set null` value.

Reply:
If by any chance the company gets deleted, this should be deleted too, otherwise it would suddenly appear in other companies

Reply:
Well, I don't think someone is able to delete a company as `restrict` ondelete will arise for sure, but it's OK this way.